projects/RPi/devices/RPi4/config/distroconfig.txt

```diff
@@ -6,3 +6,6 @@ dtoverlay=vc4-kms-v3d,cma-512
 dtoverlay=rpivid-v4l2
 disable_overscan=1
 disable_fw_kms_setup=1
+# temporarily limit framebuffer size to avoid gpu memory issues at 4kp60
+max_framebuffer_width=1920
+max_framebuffer_width=1080
```

Comment:
Just stumbled upon that. Shouldn't this be max_framebuffer_height?

Reply:
Good catch, you are right. Fortunately this doesn't have an impact on LibreELEC as it doesn't use the framebuffer at all.

A fix for the underlying issue has already been PRed https://github.com/raspberrypi/linux/pull/4421, I'll drop this workaround when it got merged